drivers/clock_control/clock_control_mcux_pcc.c

```diff
@@ -65,12 +67,12 @@ static const struct clock_control_driver_api mcux_pcc_api = {
 	.get_rate = mcux_pcc_get_rate,
 };
 
-#if defined(DT_INST_0_NXP_KINETIS_PCC_LABEL)
+#if DT_INST_NODE_HAS_PROP(0, label)
```

Comment:
This should likely be `#if DT_HAS_NODE(DT_INST(0, nxp_kinetis_pcc))`.

Reply:
why?  do we different compats for PCC?

Reply:
ah, get the point now.  Will change it over from testing `label`